tests/python/unittest/test_library_loading.py

```diff
@@ -33,6 +33,7 @@ def test_library_loading():
     elif (os.name=='nt'):
         lib = 'mylib.dll'
 
-    fname = mx.test_utils.download('https://mxnet-demo-models.s3.amazonaws.com/lib_binary/'+lib)
-    fname = os.path.abspath(fname)
+    # fname = mx.test_utils.download('https://mxnet-demo-models.s3.amazonaws.com/lib_binary/'+lib)
+    # fname = os.path.abspath(fname)
+    fname = 'Somewhere/In/The/CI'
```

Comment:
@marcoabreu what should we change this to to get the path to the library libsample_lib.so/.dll?

Reply:
Check the environment variables. I think there's one called "WORKSPACE" which points to the folder that is allocated for that job